dask_cuda/tests/test_local_cuda_cluster.py

```diff
@@ -29,8 +29,10 @@ def get_visible_devices():
                     range(utils.get_n_gpus())
                 )
 
-            # Use full memory
-            assert sum(w.memory_limit for w in cluster.workers.values()) == MEMORY_LIMIT
+            # Check workers use equal portion of full memory, ignoring FP rounding difference
+            per_worker_memory = round(MEMORY_LIMIT/utils.get_n_gpus())
+            for i in range(utils.get_n_gpus()):
+                assert cluster.workers[i].memory_limit == per_worker_memory
```

Comment:
I think this approach may still cause a similar problem in some cases. I would suggest testing for some limits instead:
```python
full_mem = sum(w.memory_limit for w in cluster.workers.values())
assert full_mem >= MEMORY_LIMIT-1024 and full_mem < MEMORY_LIMIT-1024
```

Reply:
(I think that @pentschev meant a `+` rather than a `-` in that last case.)

Reply:
Indeed, the upper limit should have a `+`, I fixed that now.

Reply:
Thank you @pentschev for suggesting an alternative approach.
The earlier approach had worked fine, and the new one also works. I updated the PR as per your suggestion. Thanks!

Reply:
Yes, the previous approach was probably good for most cases too, but having some more relaxed condition is usually more robust.